cookbook/remote_access/remote_task.py

```diff
@@ -36,4 +39,47 @@
 
     flytectl get execution -p flytesnacks -d development <execid>
 
+
+Using flytekit (python):
+========================
+More details can be found in the docs for  `FlyteRemote <https://docs.flyte.org/projects/flytekit/en/latest/remote.html>`__
+
+.. code-block:: python
+    from flytekit.remote import FlyteRemote
+    from flytekit.configuration import Config, SerializationSettings
+
+    # FlyteRemote object is the main entrypoint to API
+    remote = FlyteRemote(
+        config=Config.for_endpoint(endpoint="flyte.example.net"),
+        default_project="flytesnacks",
+        default_domain="development",
+    )
+
+    # Get Task
+    task_1 = remote.fetch_task(name="core.basic.hello_world.say_hello", version="v1")
+
+    # Tasks, workflows, and launch plans can be registered using FlyteRemote.
+    flyte_entity = ...
+    flyte_task = remote.register_task(
+        entity=flyte_entity,
+        serialization_settings=SerializationSettings(image_config=None),
+        version="v1",
+    )
+    flyte_workflow = remote.register_workflow(
+        entity=flyte_entity,
+        serialization_settings=SerializationSettings(image_config=None),
+        version="v1",
+    )
+    flyte_launch_plan = remote.register_launch_plan(entity=flyte_entity, version="v1")
+
+    # Run Task
+    execution = remote.execute(
+        task_1, inputs={...}, execution_name="my_execution", wait=True
+    )
+
+    # Inspecting execution
+    # The 'inputs' and 'outputs' correspond to the top-level execution or the workflow itself.
+    input_keys = execution.inputs.keys()
+    output_keys = execution.outputs.keys()
+
```

Comment:
@jerempy, can you also modify the flytectl instructions to include the same code path, say "workflows.descriptive_statistics" and inputs? The code in the [getting started example](https://docs.flyte.org/en/latest/getting_started/index.html) is the one I'm referring to.

Reply:
Similar instructions apply to all the other files as well. Note that `register_...` should be either task/workflows/launchplan.